src/caremote.c

```diff
@@ -197,6 +199,15 @@ CaRemote* ca_remote_unref(CaRemote *rr) {
         return mfree(rr);
 }
 
+int ca_remote_set_rate_limit(CaRemote *rr, size_t rate_limit) {
```

Comment:
should be uint64_t here, too (also: using size_t here, and int for th actual type is evil, anyway ;-))